tap_shopify/streams/base.py

```diff
@@ -1,6 +1,7 @@
 import math
 import functools
 import datetime
+from datetime import datetime, timedelta
```

Comment:
Changing this import broke code down below `datetime.timedelta(days=date_window_size)`